LiteCore/tests/VectorQueryTest.cc

```diff
@@ -573,8 +568,7 @@ TEST_CASE_METHOD(SIFTVectorQueryTest, "Index isTrained API", "[Query][.VectorSea
             store          = &db->getKeyStore(string(".") + collectionName);
         }
 
-        expectedTrained    = false;
-        expectedPretrained = false;
+        expectedTrained = false;
```

Comment:
What happened to the preTrained test?

Reply:
I removed `expectedPretrained` because Clang warned that its value is never read. I assumed it was left in the code unintentionally.

Reply:
It seems like an merge error. 7b21a6436b73ed.

Reply:
Filed a ticket for it [CBL-5983](https://issues.couchbase.com/browse/CBL-5983) f